Library/Homebrew/formula.rb

```diff
@@ -775,6 +775,8 @@ def to_hash
           "used_options" => tab.used_options.as_flags,
           "built_as_bottle" => tab.built_bottle,
           "poured_from_bottle" => tab.poured_from_bottle
+          if ENV["HOMEBREW_CXX"]: "CXX_Compiler" => ENV["HOMEBREW_CXX"]
```

Comment:
Thanks for the feedback. If I develop this idea and come back with correct syntax could it be accepted? Is there a reason brew doesn't include the CC/CXX info in the receipt?